assembl/static2/js/app/pages/debate.jsx

```diff
@@ -18,6 +21,7 @@ class Debate extends React.Component {
     this.showThumbnails = this.showThumbnails.bind(this);
     this.hideThumbnails = this.hideThumbnails.bind(this);
     this.displayThumbnails = this.displayThumbnails.bind(this);
+    this.redirectToV1Threads = this.redirectToV1Threads.bind(this);
```

Comment:
No, in the timeline, you can add many phases, so the specification is: "Each phase that has no the survey identifier => redirection to version 1"